Makefile

```
@@ -0,0 +1,7 @@
+.PHONY: post
+post:
+	@./new.sh
+
+.PHONY: taglist
+taglist:
+	@find -type f -name "index.md" | grep -v -E "^\.\/(\.\w|node_modules)" | xargs cat | grep -E "^tags: \[" | sed -r -e "s/^tags:\s\[(.*)\]$$/\1/g" | sed -r -e "s/,\s?/\n/g" | sed -r -e '/\w/!d' | sed -r -e "s/\"//g" | sort | uniq -c | column -x
```

`xargs` の `-L` オプションで 1 を指定すると、 `cat` が不要になります。
(`-L` オプションで、コマンドの引数に渡す最大の行数が指定できます。)

```suggestion
	@find -type f -name "index.md" | grep -v -E "^\.\/(\.\w|node_modules)" | xargs -L 1 grep -E "^tags: \[" | sed -r -e "s/^tags:\s\[(.*)\]$$/\1/g" | sed -r -e "s/,\s?/\n/g" | sed -r -e '/\w/!d' | sed -r -e "s/\"//g" | sort | uniq -c | column -x
```

---

ありがとうございます。これは記事ではないので修正してもらって大丈夫です。

---

承知しました。
修正して push します。